File: AutomationWithSelenium/Interfaces/Interfaces.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using OpenQA.Selenium;

namespace AutomationWithSelenium.Interfaces
{
    class Layout
    {

        public static By imgLogo = By.XPath("//img[@src='img/logo.png']");


    }

    class BetContainer
    {

        public static By btnSpin = By.Id("spinButton");
        public static By btnBetUp = By.Id("betSpinUp");
        public static By btnBetDown = By.Id("betSpinDown");
        public static By txtCredits = By.Id("credits");
        public static By txtBet = By.Id("bet");
        public static By txtLastWin = By.Id("lastWin");

    }

    class WinChartPanel
    {
        public static By tblWinChart = By.Id("prizes_list_slotMachine1");
        
    }

    class CustomizePanel
    {
        public static By btnChangeBackground = By.ClassName("btnChangeBackground");
        public static By btnChangeIcons = By.ClassName("btnChangeReels");
        public static By btnChangeMachine = By.ClassName("btnChangeMachine");
    }

}

File: Javascript/SlotMachineAutomation/Python/src/interfaces.py
class By:
    Id = staticmethod(lambda id: f"By.id('{id}')")
    ClassName = staticmethod(lambda class_name: f"By.className('{class_name}')")

tblWinChart = By.Id("prizes_list_slotMachine1")

class CustomizePanel:
    btnChangeBackground = By.ClassName("btnChangeBackground")
    btnChangeIcons = By.ClassName("btnChangeReels")
    btnChangeMachine = By.ClassName("btnChangeMachine")